src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -35,7 +35,7 @@ public void AttachPresenter(IImporterPresenter presenter)
 
       public void AddImporterView(IImporterDataView importerDataView)
       {
-         previewXtraTabControl.FillWith(importerDataView);
+         previewXtraTabControl.AddPageFrom(importerDataView, DATA_PAGE_INDEX);
```

Comment:
there are now only initialized once and not everytime the user clics

Reply:
@georgeDaskalakis Instead of overriding the hole content of the tab control, I am adding a page once